openvdb/openvdb/python/CMakeLists.txt

```diff
@@ -114,6 +114,14 @@ if(USE_NUMPY)
   list(APPEND OPENVDB_PYTHON_REQUIRED_COMPONENTS NumPy)
 endif()
 
+# scikit-build doesn't yet support FindPython, but this workaround was
+# supplied by one of the scikit-build devs. See
+# https://github.com/pypa/cibuildwheel/issues/727 for details.
+if(SKBUILD)
```

Comment:
is skbuild always guaranteed to set this CMake variable?

Reply:
It is - see [this doc](https://scikit-build.readthedocs.io/en/latest/usage.html#how-to-test-if-scikit-build-is-driving-the-compilation) for the reference.

Reply:
I think this might not be needed anymore. scikit-build-core has supported FindPython since the beginning. Also as a side note, scikit-build-core != scikit-build.